packages/components/src/List/Toolbar/Filter/Filter.component.js

```diff
@@ -30,13 +30,14 @@ function FilterInput(props) {
 		onFocus,
 		onFilter,
 		onToggle,
+		placeholder
 	} = props;
 
 	const inputProps = {
 		id,
 		name: 'search',
 		type: 'search',
-		placeholder: 'Filter',
+		placeholder: placeholder ? placeholder : 'Filter',
```

Comment:
You can use `placeholder || 'Filter'` here and use [defaultProps](https://facebook.github.io/react/docs/react-component.html#defaultprops) instead